driver/lchmod_unix.go

```diff
@@ -0,0 +1,14 @@
+// +build darwin freebsd solaris
+
+package driver
+
+import (
+	"os"
+
+	"github.com/containerd/continuity/sysx"
+)
+
+// Lchmod changes the mode of a file not following symlinks.
+func (d *driver) Lchmod(path string, mode os.FileMode) error {
+	return sysx.Fchmodat(sysx.AT_FDCWD, path, uint32(mode), sysx.AT_SYMLINK_NOFOLLOW)
```

Comment:
Can we use `golang.org/x/sys/unix` directly here? The `sysx` package is only intended to supplement which doesn't exist in that package, not wrap it

Reply:
Sure; I can remove the `Fchmodat()` from `sysx` entirely (hoping it does not have any external users).

Reply:
Updated, PTAL